server/plugin.go

```diff
@@ -145,6 +155,22 @@ func (p *Plugin) OnActivate() error {
 		return errors.WithMessage(err, "OnActivate: failed to register command")
 	}
 
+	// Create our queue of webhook events waiting to be processed.
+	bufSize, err := strconv.Atoi(p.getConfig().WebhookBufferSize)
+	if err != nil {
+		return errors.WithMessage(err, "OnActivate: error parsing config setting WebhookBufferSize")
+	}
+	p.webhookQueue = make(chan []byte, bufSize)
```

Comment:
If the admin changes the config we will leak workers because we didn't close the previous queue. We should close the previous queue if it exists before creating a new one.

Reply:
I don't think that's a problem here since we're only doing the creation in OnActivate. You would have to kill the process (killing the old channel) then restart it to get a new channel. No?

Reply:
Oh sorry. GitHub made it look like this was in OnConfigurationChange.